lib/insights/api/common/application_controller_mixins/request_body_validation.rb

```diff
@@ -38,7 +36,8 @@ def validate_request
               request.method,
               request.path,
               api_version,
-              body_params.as_json
+              body_params.to_h,
```

Comment:
I don't remember the specifics, but I think there is a difference between `as_json` and `to_h`.

Reply:
@bdunne 
The to_h returns a Hash similar to as_json but it changes the attribute values

For multipart/form-data we have a UploadedFile object that gets lost when we do the as_json

(byebug) j.to_h
{"content"=>#<ActionDispatch::Http::UploadedFile:0x00007fed85a2b438 @tempfile=#<Tempfile:/var/folders/kr/3m9v5qk557970qwndrlkdxyr0000gn/T/RackMultipart20191114-11952-19uzl2c.svg>, @original_filename="ocp_logo.svg", @content_type="image/svg+xml", @headers="Content-Disposition: form-data; name=\"content\"; filename=\"ocp_logo.svg\"\r\nContent-Type: image/svg+xml\r\nContent-Length: 21247\r\n">, "source_id"=>"27", "source_ref"=>"icon_ref", "portfolio_item_id"=>"2755"}

(byebug) j.as_json
{"content"=>{"tempfile"=>"#<File:0x00007fed7f53f1a0>", "original_filename"=>"ocp_logo.svg", "content_type"=>"image/svg+xml", "headers"=>"Content-Disposition: form-data; name=\"content\"; filename=\"ocp_logo.svg\"\r\nContent-Type: image/svg+xml\r\nContent-Length: 21247\r\n"}, "source_id"=>"27", "source_ref"=>"icon_ref", "portfolio_item_id"=>"2755"}